tests/appsec/contrib_appsec/django_app/urls.py

```diff
@@ -36,7 +36,7 @@ def multi_view(request, param_int=0, param_str=""):
         "path_params": {"param_int": param_int, "param_str": param_str},
         "query_params": query_params,
         "cookies": dict(request.COOKIES),
-        "body": request.body.decode("utf-8"),
+        "body": request.body.decode("utf-8", errors="ignore"),
```

Comment:
Do we really need this? I feel like it could hide future errors with encodings (as this change is in the test)

Reply:
yes, could be part of another PR anyway.